README.md

```diff
@@ -1,12 +1,12 @@
 ![FlagRush Logo](Assets/png/mark.png)
 
-**`FlagRush`** is a Trackmania gamemode, where two teams compete in an arena to capture a flag and bring it to opposing team base. Mode plays best with a team size of 3 players per team.
+**`FlagRush`** is a Trackmania gamemode, where two teams compete in an arena to capture a flag and bring it to opposing team base. Mode plays best with a team size of 4 players per team.
 
 # Maps
 
 We recommend starting with our officially reviewed mapppacks:
-- 3v3 Mappack: https://trackmania.exchange/mappack/view/1155/flagrush-3v3-map-pool
-- 5v5 Mappack: https://trackmania.exchange/mappack/view/1156/flagrush-5v5-map-pool
+- Curated Mappack: https://trackmania.exchange/mappack/view/1155/flagrush-curated-map-pool
+- Proving Grounds Mappack: https://trackmania.exchange/mappack/view/1151/flagrush-proving-grounds
```

Comment:
I think we could describe what these mappacks are for, to mention that proving grounds maps are 'in review' while curated maps are recommended